aspnetcore/security/data-protection/extensibility/key-management.md

```diff
@@ -126,7 +126,7 @@ The `IXmlRepository` interface represents a type that can persist XML to and ret
 
 Implementations of `IXmlRepository` don't need to parse the XML passing through them. They should treat the XML documents as opaque and let higher layers worry about generating and parsing the documents.
 
-There are two built-in concrete types which implement `IXmlRepository`: `FileSystemXmlRepository` and `RegistryXmlRepository`. See the [key storage providers document](xref:security/data-protection/implementation/key-storage-providers#data-protection-implementation-key-storage-providers) for more information. Registering a custom `IXmlRepository` would be the appropriate manner to use a different backing store, e.g., Azure Blob Storage.
+There are two built-in concrete types which implement `IXmlRepository`: `FileSystemXmlRepository` and `RegistryXmlRepository`. See the [key storage providers document](xref:security/data-protection/implementation/key-storage-providers) for more information. Registering a custom `IXmlRepository` would be the appropriate manner to use a different backing store, e.g., Azure Blob Storage.
```

Comment:
Mention the Redis repository?

Reply:
Yes, this section is dated. I'll update with the current full list ...

Microsoft.AspNetCore.DataProtection.RedisXmlRepository
Microsoft.AspNetCore.DataProtection.AzureStorage.AzureBlobXmlRepository
Microsoft.AspNetCore.DataProtection.Repositories.FileSystemXmlRepository
Microsoft.AspNetCore.DataProtection.Repositories.RegistryXmlRepository